### DISCUSSION TITLE
After registering  Account  is disabled.(Account suspended)

### DISCUSSION THREAD

**Discussion by caozg5** [community/community#143938] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

After registering  Account  is disabled.(Account suspended)，how restore or unbind the email？

**Comment by bipanshu-kumar** [comment]
is it a genuine question or you're just testing something??

**Reply by caozg5** [reply]
It was a problem that occurred when my colleague registered using an email, and it was disabled after registration

**Comment by Cardosofiles** [comment]
If your GitHub account was disabled or suspended after registration, you can try the following steps to restore it or unbind the associated email:

Review the Account Suspension Notice: GitHub typically sends an email explaining the reason for the account suspension. Check your inbox (including the spam/junk folder) for any communication from GitHub.

Contact GitHub Support:

Go to [GitHub's Support Contact Page](https://support.github.com/).
Choose the option related to "Account" issues or "Suspended/Disabled Account."
Provide your GitHub username and the email associated with the account, and briefly explain the issue.
Request for your account to be reviewed or for the email address to be unbound from the disabled account if restoration isn’t possible.
Check GitHub’s Terms of Service: If your account was disabled due to a violation, review GitHub’s [Terms of Service](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service) to understand what went wrong.

Wait for a Response: GitHub Support may take some time to review and respond to your request. Be patient, as account suspensions are handled manually to ensure accurate resolution.

If GitHub Support allows it, they may restore your account or unbind the email so you can use it with a new account.

**Reply by caozg5** [reply]
Already submitted [Tickets]（ https://support.github.com/tickets/personal/0 ）However, there has been no response

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @caozg5!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you. Good luck!
